evm/deploy-gateway-v6.2.x.js

```diff
@@ -368,6 +368,8 @@ async function deploy(config, chain, options) {
     contractConfig.tokenDeployer = tokenDeployer.address;
     contractConfig.deployer = wallet.address;
     contractConfig.deploymentMethod = options.deployMethod;
+    contractConfig.connectionType = options.amplifier ? 'amplifier' : 'consensus';
+    contractConfig.chainType = 'evm';
 
```

Comment:
update `mainProcessor` util to only loop over all chains that have `chainType` === 'evm'. can do this in a separate PR